src/navigation-plan.js

```diff
@@ -1,8 +1,10 @@
 import {Redirect} from './navigation-commands';
 
-export var NO_CHANGE = 'no-change';
-export var INVOKE_LIFECYCLE = 'invoke-lifecycle';
-export var REPLACE = 'replace';
+export const activationStrategy = {
```

Comment:
This is named like an enum type, but maybe the plural `activationStrategies` would be more appropriate.